doc/source/user_guide/tutorials/animate/animate_data.rst

```diff
@@ -0,0 +1,503 @@
+location (only elemental, nodal or faces location are supported for the animation).
+For more information on how to define a scoping check the ``Narrow down data`` tutorial in the :ref:`ref_tutorials_import_data`
+tutorials section.
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+        .. code-block:: python
+
+            # Get all the time steps
+            time_scoping_1 = my_model_1.metadata.time_freq_support.time_frequencies
+            # Get all the mesh in a nodal location
+            mesh_scoping_1 = dpf.Scoping(ids=my_meshed_region_1.nodes.scoping.ids, location=dpf.locations.nodal)
+
+    .. tab-item:: LSDYNA
+
+        .. code-block:: python
+
+            # Get all the time steps
+            time_scoping_2 = my_model_2.metadata.time_freq_support.time_frequencies
+
+Extract the results
+-------------------
+
+You have to extracted the results of interest.
+
+When you animate the data you go through the |Fields| of a |FieldsContainer| and plot contours of
+the data norm or of the selected data component. This means that the geometry needs to be deformed
+based on the |Fields| themselves.
+
+The geometry can be deformed by a |Result| object, an |Operator| (It must evaluate to a FieldsContainer
+of same length as the one being animated), a |Field| or a |FieldsContainer|.
+
+To deform the geometry we need a result with a homogeneous unit dimension, thus, a distance unit.
+Thus, to deform the mesh we need the displacement result.
+For more information see: :ref:`ref_plotting_data_on_deformed_mesh`
+
+Here we get:
+
+- The displacement results and the stress result for the MAPDL result file
+- The displacement results and the beam axial force result for the LSDYNA result file
+
+.. tab-set::
+
+    .. tab-item:: MAPDL
+
+        .. code-block:: python
+
+            # Get the displacement results
+            my_disp_1 = my_model_1.results.displacement(time_scoping=time_scoping_1,
+                                                        mesh_scoping=mesh_scoping_1).eval()
+            # Get the stress results
+            my_stress_1 = my_model_1.results.stress(time_scoping=time_scoping_1,
+                                                        mesh_scoping=mesh_scoping_1).eval()
+
+    .. tab-item:: LSDYNA
+
+        .. code-block:: python
+
+            # Get the displacement results
+            my_disp_2 = my_model_2.results.displacement(time_scoping=time_scoping_2.eval()
+            # Get the stress results
+            my_beam_axial_force_2 = my_model_2.results.beam_axial_force(time_scoping=time_scoping_2).eval()
+
+Animate the results
+-------------------
+
+You animate a |FieldsContainer| by using the |animate| method.
+
+The default behavior consists in:
+
+- Using a constant and uniform scale factor of 1.0
+- Showing the deformed geometry if the method was used directly with the displacement fields.
+- Showing the static geometry if the method was used with other results fields.
```

Comment:
```suggestion
- Showing the deformed geometry if the method is used directly with the displacement fields.
- Showing the static geometry if the method is used with other results fields.
```

Reply:
Check about the default behavior for a displacement field/3D vector field. We should be able to animate it without meh deformation and explain it here.